lib/trino-orc/src/test/java/io/trino/orc/OrcTester.java

```diff
@@ -976,21 +979,15 @@ public static void writeOrcColumnHive(RecordWriter recordWriter, Type type, Iter
             throws Exception
     {
         SettableStructObjectInspector objectInspector = createSettableStructObjectInspector("test", type);
-        Object row = objectInspector.create();
-
-        List<StructField> fields = ImmutableList.copyOf(objectInspector.getAllStructFieldRefs());
-        Serializer serializer = new OrcSerde();
-
-        while (values.hasNext()) {
-            Object value = values.next();
-            value = preprocessWriteValueHive(type, value);
-            objectInspector.setStructFieldData(row, fields.get(0), value);
-
-            Writable record = serializer.serialize(row, objectInspector);
-            recordWriter.write(record);
-        }
-
-        recordWriter.close(false);
+        writeOrcColumnsHive(
+                recordWriter,
+                (StandardStructObjectInspector) objectInspector,
```

Comment:
why not use `SettableStructObjectInspector` as a parameter for `writeOrcColumnsHive` ?

Reply:
done